tests/e2e/utils.js

```diff
@@ -409,24 +410,18 @@ export const createCoupon = async ( coupon ) => {
 };
 
 /**
- * Open the block editor settings menu.
- *
- * @param {Object}  [root0]
- * @param {boolean} [root0.isFSEEditor] Amount to be applied. Defaults to 5.
+ * Open the block editor settings menu if it hasn't opened.
  */
+export const openBlockEditorSettings = async () => {
+	const toggleSidebarButton = await page.$(
+		'.edit-post-header__settings [aria-label="Settings"][aria-expanded="false"],' +
+			'.edit-site-header__actions [aria-label="Settings"][aria-expanded="false"],' +
+			'.edit-widgets-header__actions [aria-label="Settings"][aria-expanded="false"],' +
+			'.edit-site-header-edit-mode__actions [aria-label="Settings"][aria-expanded="false"]'
```

Comment:
Would just `[aria-label="Settings"][aria-expanded="false"]` not be enough to cover all of those cases? Or is it liable to be confused with some other selector?

Reply:
I'm afraid `[aria-label="Settings"][aria-expanded="false"]` might be too generic. Using 'header' classes is safer IMO. I created a [PR](https://github.com/WordPress/gutenberg/pull/45480) updating the `ensureSidebarOpened` upstream. If it's merged, we can then use `ensureSidebarOpened` instead.

Reply:
[My PR](https://github.com/WordPress/gutenberg/pull/45480) updated `ensureSidebarOpened` has been merged.

Reply:
Since this has been merged, can we get rid of this and use your util on Gutenberg?

Reply:
We need to wait until Gutenberg releases a new version of `@wordpress/e2e-test-utils`. I added a to-do to explain the reason and keep track of that refactoring task.